### PR TITLE
feature: storage bucket init version

### PR DESCRIPTION
**Pull request by karolcienkosz**

### CODE REVIEW

**Review by michalbagrowski — DISMISSED** [review]

**Review by michalbagrowski — COMMENTED** [review]
add .DS_Store to ignored files

**Review by michalbagrowski — COMMENTED** [review]
why emoty notes file?

**Comment by karolcienkosz** [comment]
README has been removed and .gitignore has been added into separate PR

**Review by michalbagrowski — DISMISSED** [review]

File: charts/storage-bucket-config-connector/templates/storage-bucket.yaml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    enabled: {{ required ".autoclass.enabled is required" $bucket.spec.autoclass.enabled }}
+    {{- else }}
+    enabled: {{ required ".autoclass.enabled is required" $.Values.default.spec.autoclass.enabled }}
+    {{- end }}
+  {{- end }}
+  bucketPolicyOnly: {{ $bucket.spec.bucketPolicyOnly | default $.Values.default.spec.bucketPolicyOnly }}
+  {{- if or $bucket.spec.cors $.Values.default.spec.cors }}
+  cors:
+  {{- if $bucket.spec.cors }}
+  {{- range $cor := $bucket.spec.cors  }}
```
Comment (mirkoszy) [inline charts/storage-bucket-config-connector/templates/storage-bucket.yaml]: cor? cors is not a plural of cor.

Comment (karolcienkosz) [reply charts/storage-bucket-config-connector/templates/storage-bucket.yaml]: right!. fixed

**Review by michalbagrowski — APPROVED** [review]

**Pull request merged**